certbot/docs/using.rst

```diff
@@ -555,6 +555,11 @@ Deleting certificates
 
 If you need to delete a certificate, use the ``delete`` subcommand.
 
+Certbot does not automatically revoke a certificate before deleting it. If you're no longer using a certificate and don't
+plan to use it anywhere else, you may want to follow the instructions in `Revoking certificates`_ first. Generally, there's
```

Comment:
```suggestion
plan to use it anywhere else, you may want to follow the instructions in `Revoking certificates`_ instead. Generally, there's
```

Calling `revoke` will prompt the user to delete as well. Since this is the case, this added section should also probably go after the note directly below it, so we're sure the user has read the "safely deleting certificates" section, because if they are helped by these instructions they will probably also want to delete the certificate.

Reply:
Yep, great point! I've pushed a commit for it